<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BitsWidget</class>
 <widget class="QWidget" name="BitsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>49</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBoxBit0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxBit7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
